packages/page-bounties/src/Voting/VotingLink.tsx

```diff
@@ -0,0 +1,53 @@
+// Copyright 2017-2021 @polkadot/app-bounties authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+import styled from 'styled-components';
+
+import { Icon } from '@polkadot/react-components';
+
+import Description from '../Description';
+import { useTranslation } from '../translate';
+
+interface Props {
+  className?: string;
+}
+
+function VotingLink ({ className }: Props): React.ReactElement<string> {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <div className={className}>
+        <a className='voting-link'
+          href='#/council/motions'
+        >
+          <b>{t('Voting')}</b>
+          <Icon
+            className='voting-icon'
+            icon='chevron-right'
+          />
+        </a>
+        <Description description={t<string>('Go to motions panel')} />
+      </div>
+    </>
+  );
+}
+
+export default React.memo(styled(VotingLink)`
+  && .voting-link {
+    font-size: 0.85rem;
+    line-height: 0.5rem;
+    color: currentColor;
+    :hover {
+      color: currentColor;
+    }
+  }
```

Comment:
The links should just be links (as it is everywhere atm) - the button-like appearance imho probably drives away from "open-in-new-window".